runtime/launcher.m

```diff
@@ -495,14 +497,9 @@
 		NSString *machine_config_path = nil;
 		NSError *error = nil;
 		
-#ifdef DYNAMIC_MONO_RUNTIME
-		config_path = [[NSString stringWithUTF8String: mono_runtime_prefix] stringByAppendingPathComponent: @"/etc/mono/config"];
-#else
+#ifndef DYNAMIC_MONO_RUNTIME
```

Comment:
This looks good, but can you also update the comment just above, and the one here: https://github.com/xamarin/xamarin-macios/blob/bcd2cc2d94b624de83e91da08f3bc26e66224776/runtime/xamarin/launch.h#L75, so something like this:

```csharp
// 7a) [If not embedding] Don't parse any config files, leave it to the Mono runtime to load whatever it needs.
```

Reply:
Modified comment to `Don't parse any config files, leave it to the Mono runtime to load the defaults.`, and modified it in xamarin/launch.h too